docs/attributes-registry/cicd.md

```diff
@@ -28,3 +44,25 @@ This group describes attributes specific to pipelines within a Continuous Integr
 | `build` | build | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `deploy` | deploy | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `test` | test | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+---
+
+`cicd.worker.state` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `busy` | The worker is performing work for the CICD system. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `down` | The worker is not available to the CICD system (disconnected / down). | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `idle` | The worker is not performing work for the CICD system. It is available to the CICD system to perform work on. [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+**[1]:** Pipelines might have conditions on which workers they are able to run so not every worker might be available to every pipeline.
+
+---
+
+`cicd.worker.type` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
```

Comment:
In many complex CI/CDs the workers are categorized by other attributes, such as the platform they run (`ubuntu`, `windows`) the tooling they have installed  (`java`, `go`) or other attributes. 

I lean towards leaving this attribute open (without pre-defined values). Also consider `class` as an alternative name to reflect its intention to group the workers into categories.

Reply:
I have rename this attribute to `cicd.worker.class` in 26d3064808bb0ead5fcd0db81af2fa04820f696a.